roles/installer/templates/deployments/web.yaml.j2

```diff
@@ -137,6 +138,15 @@ spec:
           securityContext:
             capabilities:
               add: {{ redis_capabilities }}
+{% endif %}
+{% if redis_host_network %}
+          hostNetwork: {{ redis_host_network }}
+{% endif %}
+{% if redis_dns_policy %}
+         dnsPolicy: {{ redis_dns_policy }}
+{% endif %}
+{% if redis_dns_policy == 'None' and redis_dns_config is defined }}
```

Comment:
Similarly, there is an extra closing brace at the end of this line too which causes syntax errors when applied.